packages/firestore/src/core/firestore_client.ts

```diff
@@ -291,32 +302,37 @@ export class FirestoreClient {
     });
 
     return Promise.resolve().then(() => {
-      this.persistence = new IndexedDbPersistence(
+      const persistence = new IndexedDbPersistence(
```

Comment:
nit: Not sure why you introduced `persistence` separate from `this.persistence`

Reply:
this.persistence is of type Persistence, which doesn't expose `synchronizeTabs`. I made this more obvious by adding the type to `const persistence`.